### PR TITLE
modified ddldump to support sequences

### PR DESCRIPTION
**Pull request by dimitrylukashov**
JIRA: https://percolate.atlassian.net/browse/IG-199

### CODE REVIEW

**Comment by dimitrylukashov** [comment]
@lra how do we disable the flake8 test? Phaser is blocking the rest of the build, and its not really adding much value.

**Comment by dimitrylukashov** [comment]
@lra I replaced `flake8` with `black` and rewrote some of the other tests. Can change cci to not expect flake8 anymore?

@chhsiung when you get a chance, can you take a look?

File: ddldump/main.py [hunk 1]
```diff
@@ -197,22 +197,43 @@ def show_create(self, table):
...
 
+        # Separate CREATE TABLE STATEMENT
         # Separating the CREATE TABLE statement and the rest of the details
         # from pg_dump output for better manipulation.
-        raw_output_less_create_table = raw_output.replace(table_ddl_create, "")
+        create_table_start = raw_output[raw_output.find("CREATE TABLE") :]
+        create_table_statement = create_table_start[: create_table_start.find(";") + 1]
+        raw_output_without_table = raw_output.replace(create_table_statement, "")
+
+        # Seprate CREATE SEQUENCE STATEMENTS
```
Comment (lra) [inline ddldump/main.py]: separate

Comment (lra) [reply ddldump/main.py]: https://github.com/percolate/ddldump/pull/19/commits/61e3191f629324a8f05a3b3de9449669a70b6e4e

**Comment by lra** [comment]
https://pypi.org/project/ddldump/

**Pull request merged**